modules/osdk-updating-131-to-1361.adoc

```diff
@@ -0,0 +1,289 @@
+.Example Makefile
+[source,make,subs="attributes+"]
+----
+# Set the Operator SDK version to use. By default, what is installed on the system is used.
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v{osdk_ver}-ocp
+----
+
+. Update the `kube-rbac-proxy` container to use the {op-system-base-full} 9-based image:
+
+.. Find the entry for the `kube-rbac-proxy` container in the following files:
++
+--
+* `config/default/manager_auth_proxy_patch.yaml`
+* `bundle/manifests/<operator_name>.clusterserviceversion.yaml` for your Operator project, for example `memcached-operator.clusterserviceversion.yaml` from the tutorials
+--
+
+.. Update the image name in the pull spec from `ose-kube-rbac-proxy` to `ose-kube-rbac-proxy-rhel9`, and update the tag to `v{product-version}`:
++
+.Example `ose-kube-rbac-proxy-rhel9` pull spec with `v{product-version}` image tag
+[source,yaml,subs="attributes+"]
+----
+# ...
+      containers:
+      - name: kube-rbac-proxy
+        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy-rhel9:v{product-version}
+# ...
+----
+
+ifdef::helm[]
+. Edit your Operator's Dockerfile to update the `ose-helm-rhel9-operator` image tag to `{product-version}`, as shown in the following example:
++
+.Example Dockerfile
+[source,docker,subs="attributes+"]
+----
+FROM registry.redhat.io/openshift4/ose-helm-rhel9-operator:v{product-version}
+----
+endif::[]
+ifdef::ansible[]
+. Edit your Operator's Dockerfile to update the `ose-ansible-rhel9-operator` image tag to `{product-version}`, as shown in the following example:
++
+.Example Dockerfile
+[source,docker,subs="attributes+"]
+----
+FROM registry.redhat.io/openshift4/ose-ansible-rhel9-operator:v{product-version}
+----
+endif::[]
+ifdef::golang,hybrid[]
+. The `go/v4` plugin is now stable and is the default version used when scaffolding a Go-based Operator. The transition from Golang v2 and v3 plugins to the new Golang v4 plugin introduces significant changes. This migration is designed to enhance your project's functionality and compatibility, reflecting the evolving landscape of Golang development.
++
+For more information on the reasoning behind these changes, see link:https://book.kubebuilder.io/migration/v3vsv4#tldr-of-the-new-gov4-plugin[go/v3 vs go/v4] in the Kubebuilder documentation.
++
+For a comprehensive understanding of the migration process to the v4 plugin format and detailed migration steps, see link:https://book.kubebuilder.io/migration/manually_migration_guide_gov3_to_gov4[Migration from go/v3 to go/v4 by updating the files manually] in the Kubebuilder documentation.
+endif::[]
+
+. The `kustomize/v2` plugin is now stable and is the default version used in the plugin chain when using `go/v4`, `ansible/v1`, `helm/v1`, and `hybrid/v1-alpha` plugins. For more information on this default scaffold, see link:https://book.kubebuilder.io/plugins/kustomize-v2[Kustomize v2] in the Kubebuilder documentation.
```

Comment:
the Kustomize v2 link appears to go to a non-functional page now

Reply:
Oh no!

Reply:
OK thanks, looks like it's https://book.kubebuilder.io/plugins/available/kustomize-v2 now.